tests/app/questionnaire/test_questionnaire_store_updater.py

```diff
@@ -379,7 +383,7 @@ def test_remove_completed_relationship_locations_for_list_name(
     patch_method = "app.questionnaire.questionnaire_store_updater.QuestionnaireStoreUpdater._get_relationship_collectors_by_list_name"
     patched = mocker.patch(patch_method)
     patched.return_value = [{"id": "test-relationship-collector"}]
-    questionnaire_store_updater.remove_completed_relationship_locations_for_list_name(
+    questionnaire_store_updater._remove_completed_relationship_locations_for_list_name(  # pylint: disable=protected-access
```

Comment:
These methods are dedicated to testing this now private method, I suggest for now make it public instead of disabling linting as we shouldn't be testing private method. You could remove it but would mean another test is needed but that is outside the scope of this PR so revert this one to public for now.

Reply:
I updated some of the methods to private cause otherwise the updater method gets the too may public methods linting failure

Reply:
Ya, Im suggesting since this has a test (which should only be done for public method) revert it. Otherwise, we would need to rewrite the test that does go direct via the private method. Will revert just this cause the failure, are there others you can make private?

Reply:
I was able to revert this without triggering the linting error